src/gluonts/zebras/_time_frame.py

```diff
@@ -303,6 +303,57 @@ def as_dict(self, prefix=None, static=True):
 
         return result
 
+    def rolsplit(
+        self,
+        index,
+        future_length,
+        *,
+        past_length=None,
+        distance=None,
+        n=None,
+        pad_value=0.0,
+    ):
+        """Create rolling split of past/future pairs.
+
+        Parameters
+        ----------
+        index
+            Starting index that denominates the cut off point from which splits
+            are generated.
+        future_length
+            All pairs share the same `future_length` and therefore the
+            parameter is required unlike in `.split`.
```

Comment:
Is this a necessary constraint?

Reply:
I guess not. One could also set `distance`?